controller:
  ros__parameters:
    apf_gain: 0.8
    inter_agent_distance: 0.5
    segments_topic: segments
    instructions_topic: instructions
    detected_objects_topic: detected_objects
    tracked_objects_topic: tracked_objects